src/test/java/com/google/javascript/clutz/aliased_enums.d.ts

```diff
@@ -14,7 +14,7 @@ declare module 'goog:nested.bar.Enum' {
 declare namespace ಠ_ಠ.clutz_internal.nested.baz {
   type Enum = number ;
   var Enum : {
-    A : Enum ,
+    A : ಠ_ಠ.clutz_internal.nested.baz.Enum ,
```

Comment:
in this case you can guarantee the `Enum` is what you want unambiguously, but then again I don't see much value in producing the cleanest .d.ts possible.